mtags/src/main/scala-3/scala/meta/internal/pc/completions/KeywordsCompletions.scala

```diff
@@ -23,6 +23,9 @@ object KeywordsCompletions:
           case kw if (kw.isPackage || kw.isTemplate) && notInComment =>
             CompletionValue.keyword(kw.name, kw.insertText)
         }
+      case Select(qual, name) :: (appl: Apply) :: _
+          if "match".startsWith(name.toString()) =>
+        List(CompletionValue.keyword("match", "match {\n\tcase $0\n}"))
```

Comment:
I think we should suggest optional braces if possible, since match case is already very much in style of Scala 3
```suggestion
        List(CompletionValue.keyword("match", "match {\n\tcase $0\n}"))
```